pkg/grouper/grouper.go

```diff
@@ -20,6 +21,26 @@ func hasAliasIntersection(v1, v2 IDAliases) bool {
 	return slices.Contains(v1.Aliases, v2.ID) || slices.Contains(v2.Aliases, v1.ID)
 }
 
+func sortIDs(groupIDs []string) []string {
```

Comment:
Please use the existing `internal/output/identifiers.go` sort functions.  I think it's a good idea to sort it in the grouper directly like you have it here, so can you try moving the sort functions out into it's own internal package, and make it public?

Reply:
Updated.